data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon','2020-02-03', 0, TRUE, 10.23);
```

Comment:
- [ ] [OPTIONAL] Please don't forget to add the `MIT.md` license file as linked in your readme. You can get the template from [here 👉🏼](https://github.com/microverseinc/readme-template/blob/master/MIT.md)

<img width="674" alt="image" src="https://user-images.githubusercontent.com/69011963/177227755-38286502-1c20-4f36-beee-32ddc229785b.png">